CONTRIBUTING.md

```diff
@@ -22,13 +22,14 @@ From a separate terminal, start running a flutter app to connect to DevTools:
 - `cd gallery`
 - ensure the iOS Simulator is open (or a physical device is connected)
 - `flutter run`
+- open the link 'The Flutter DevTools debugger and profiler...' in the command output 
```

Comment:
Opening this link will actually open the version of DevTools that is shipped in the SDK, not the local DevTools build you are developing. Instead, add something like this or similar:

- copy the "Observatory debugger and profiler" uri printed in the command output (e.g. "http://127.0.0.1:50973/0ih7btPVPD0=/" from "An Observatory debugger and profiler on iPhone 13 is available at: http://127.0.0.1:50973/0ih7btPVPD0=/"). You will paste this uri into the Connect dialog of the local DevTools you are developing later.

Reply:
done